install.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Install Karabiner if it isn't installed, but homebrew and cask are.
+if [[ "$(type -P brew)" && "$(brew ls --versions brew-cask)" ]]; then
+  if [[ ! "$(brew cask list 2>/dev/null | grep karabiner)" ]]; then
+    echo "Installing Karabiner..."
+    brew cask install karabiner
+  fi
+fi
```

Comment:
Could you warn about missing brew or cask and maybe add install instructions for them?